Teams Gallery: make teams requires requests package to be pre-installed

Ref #510 

I used a fresh new pyenv virtualenv to run teams target.  When I run `make teams` target to build teams gallery,  it requires python requests package.  There is no requirements.txt in NumPy.org repo anymore.  Do we need to state this in README or am I missing something?

After installing requests package in my virtualenv, it worked fine.

>>>>Log for reference

BEFORE:
numpy.org % pip3 list
Package    Version
---------- -------
pip        21.2.3
setuptools 57.4.0

numpy.org % make teams
Submodule 'themes/scientific-python-hugo-theme' (https://github.com/scientific-python/scientific-python-hugo-theme.git) registered for path 'themes/scientific-python-hugo-theme'
Cloning into '/<PATH>/ssnumpyfork/numpy.org/themes/scientific-python-hugo-theme'...
Submodule path 'themes/scientific-python-hugo-theme': checked out '1ee3eb51cb7a9496bac2432b99ed73cb734c5609'
Submodule 'themes/hugo-fresh' (https://github.com/StefMa/hugo-fresh.git) registered for path 'themes/scientific-python-hugo-theme/themes/hugo-fresh'
Cloning into '<PATH>/ssnumpyfork/numpy.org/themes/scientific-python-hugo-theme/themes/hugo-fresh'...
Submodule path 'themes/scientific-python-hugo-theme/themes/hugo-fresh': checked out '17f51372b615e3c92d44d79f1edfd2fd1866aec7'
Traceback (most recent call last):
  File "<PATH>/ssnumpyfork/numpy.org/themes/scientific-python-hugo-theme/tools/team_query.py", line 6, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
make: *** [static/gallery/emeritus-maintainers.md] Error 1

AFTER installing requests package (and other dependencies automatically), it worked:


numpy.org % pip3 list
Package            Version
------------------ ---------
certifi            2021.10.8
charset-normalizer 2.0.10
idna               3.3
pip                21.2.3
requests           2.27.1
setuptools         57.4.0
urllib3            1.26.7


numpy.org % make teams
Teams dir: emeritus-maintainers
Teams dir: maintainers
Teams dir: triage-team
Teams dir: survey-team
Teams dir: web-team

--- a/README.md
+++ b/README.md
@@ -75,6 +75,8 @@ if you don't have the Hugo extended version.
 ## Deployment
 
 Submit pull requests first, those get run on [Netlify](https://quansight-labs.netlify.app/) and you can see a build preview by clicking on the `details` link at the bottom.
+ 
+![Build previews](images/readme-build-previews.png)
 
 ## Team lists
 
@@ -88,5 +90,3 @@ To update the teams gallery in numpy.org site, you need to run `make teams` Make
 export GH_TOKEN=xxxxxxxxxx
 make teams
 ```
- 
-![Build previews](images/readme-build-previews.png)

--- a/README.md
+++ b/README.md
@@ -76,17 +76,17 @@ if you don't have the Hugo extended version.
 
 Submit pull requests first, those get run on [Netlify](https://quansight-labs.netlify.app/) and you can see a build preview by clicking on the `details` link at the bottom.
 
-![Build previews](images/readme-build-previews.png)
-
-
 ## Team lists
 
-To update the teams, the `team_query.py` provided by the theme is used.
-It needs a
-[GitHub token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
-with `read:org` permissions. The token has to be exported as `GH_TOKEN`.
+To update the teams gallery in numpy.org site, you need to run `make teams` Makefile target. It uses the `team_query.py` tool provided by [scientific-python-hugo-theme](https://github.com/scientific-python/scientific-python-hugo-theme). The following pre-requisites need to be met in numpy.org build environment:
+
+* The `team_query.py` tool requires python requests package. Make sure it is installed in your numpy.org build environment before invoking `make teams` Makefile target.
+
+* [GitHub token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) with `read:org` permissions is required for requesting numpy.org team data from GitHub. The token has to be exported as `GH_TOKEN`.
 
 ```
 export GH_TOKEN=xxxxxxxxxx
 make teams
 ```
+ 
+![Build previews](images/readme-build-previews.png)

--- a/README.md
+++ b/README.md
@@ -76,17 +76,17 @@ if you don't have the Hugo extended version.
 
 Submit pull requests first, those get run on [Netlify](https://quansight-labs.netlify.app/) and you can see a build preview by clicking on the `details` link at the bottom.
 
+![Build previews](images/readme-build-previews.png)
+
+
 ## Team lists
 
-To update the teams gallery in numpy.org site, you need to run `make teams` Makefile target. It uses the `team_query.py` tool provided by [scientific-python-hugo-theme](https://github.com/scientific-python/scientific-python-hugo-theme). The following pre-requisites need to be met in numpy.org build environment:
-
-* The `team_query.py` tool requires python requests package. Make sure it is installed in your numpy.org build environment before invoking `make teams` Makefile target.
-
-* [GitHub token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) with `read:org` permissions is required for requesting numpy.org team data from GitHub. The token has to be exported as `GH_TOKEN`.
+To update the teams, the `team_query.py` provided by the theme is used.
+It needs a
+[GitHub token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+with `read:org` permissions. The token has to be exported as `GH_TOKEN`.
 
 ```
 export GH_TOKEN=xxxxxxxxxx
 make teams
 ```
- 
-![Build previews](images/readme-build-previews.png)

